funflow-examples/package.yaml

```diff
@@ -18,6 +18,7 @@ dependencies:
   - directory
   - funflow
   - kernmantle
+  - kernmantle-caching
```

Comment:
We should be able to drop this dependency if we swap the import for `caching` above.